timeseries/src/autogluon/timeseries/models/chronos/model.py

```diff
@@ -264,7 +264,7 @@ def default_batch_size(self) -> int:
         """Default batch size used for the model. For models not defined in AutoGluon, this value
         defaults to 8.
         """
-        return self.ag_default_config.get("default_batch_size", 8)
+        return self.ag_default_config.get("default_batch_size", 16)
```

Comment:
Why this change? Also appears to be different from the docstring.

Reply:
changed back 👍